daemon/oci_darwin.go

```diff
@@ -0,0 +1,109 @@
+// +build darwin
```

Comment:
(I couldn't find where to comment for the commit message, so putting here)

- what does "initial build" mean ? (OCI features required for the initial build of docker engine in darwin)
- how does `graphdriver` related to this commit ?

Reply:
> what does "initial build" mean ? (OCI features required for the initial build of docker engine in darwin)

Ah, I will rephrase that. 

> how does graphdriver related to this commit ?
 
VFS is the graphdriver that is being utalised here. And it wasn't able to get the rootfs path due to the OCI changes we had to do. Right?

Reply:
If somebody who doesn't know about your history with the graphdriver , the commit message doesn't make sense since the patch doesn't  contain any graphdriver/vfs related code.